antora-ui-camel/src/partials/footer-content.hbs

```diff
@@ -1,4 +1,4 @@
-    <footer>
+  <footer>
```

Comment:
Here, earlier I had put the <div class="top".. . Now I have removed it and for the Documentation pages, as mentioned previously, the Back to Top tag is put in the toolbar within the same lines as Edit The Page and Wide/Narrow.